src/modules/musig/main_impl.h

```diff
@@ -206,7 +191,11 @@ int secp256k1_musig_session_init(const secp256k1_context* ctx, secp256k1_musig_s
         secp256k1_scalar_clear(&secret);
         return 0;
     }
-    secp256k1_musig_coefficient(&mu, session->pre_session.pk_hash, (uint32_t) my_index);
+
+    secp256k1_ecmult_gen(&ctx->ecmult_gen_ctx, &pj, &secret);
+    secp256k1_ge_set_gej(&p, &pj);
+    secp256k1_fe_normalize_var(&p.x);
+    secp256k1_musig_coefficient(&mu, session->pre_session.pk_hash, &p.x);
```

Comment:
Maybe add more comment to ease understanding from paper "Description" section, like :
```
/* Compute the signer's coefficient from its public key point and the hashed list of cosigner's public keys
 * mu_i = H("MuSig coefficient, L, X_i) where L is the ordered list of pubkeys"
 */